tensorboard/pip_package/requirements_dev.txt

```diff
@@ -21,6 +21,8 @@ pandas~=1.0
 # For gfile S3 test
 boto3==1.9.86
 moto==1.3.7
+# For gfile fsspec test
+fsspec==2021.8.1
```

Comment:
Would you mind changing this to 0.7.4 for now? As Nick indicated, we internally are using older version of fsspec and don't want to inadvertently use a newer APIs.